client/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@hookform/error-message": "^2.0.1",
     "@tanstack/react-query": "^4.22.0",
+    "axios": "^1.3.4",
```

Comment:
remove axios and keep using fetch

Reply:
I was having a lot of issues with fetch, even when I got the request to be accepted I couldn't retrieve the same data as I do with axios. 

I don't mind swapping back, but I need help with setting fetch up properly.